packages/geolocation/src/domains/geolocation.js

```diff
@@ -0,0 +1,19 @@
+const connection = require('../drivers/mongodb/connection');
+
+const geoNearQuery = async (lon, lat, maxDistance = 1000) => {
```

Comment:
@Edilberto-Vazquez we should try to apply the inversion dependency principle here in order to reduce the coupling and this will facilitate the testing, so in order toa achieve that, we can do something like this:
```suggestion
const geoNearQuery = (connection) => async (lon, lat, maxDistance = 1000) => {
```
The example above applies a closure to get that connection applying the dependency injection principle.